autotx/AutoTx.py

```diff
@@ -147,7 +186,7 @@ def try_run(self, prompt: str, non_interactive: bool, summary_method: str = "las
 
         chat_history = json.dumps(chat.chat_history, indent=4)
 
-        return RunResult(chat.summary, chat_history, transactions, EndReason.TERMINATE if is_goal_supported else EndReason.GOAL_NOT_SUPPORTED)
+        return RunResult(chat.summary, chat_history, transactions, EndReason.TERMINATE if is_goal_supported else EndReason.GOAL_NOT_SUPPORTED, float(chat.cost["usage_including_cached_inference"]["total_cost"]), float(chat.cost["usage_excluding_cached_inference"]["total_cost"]))
```

Comment:
when testing this, it has the following error at the end:
```
Traceback (most recent call last):
  File "/home/cesar/.cache/pypoetry/virtualenvs/autotx-qxA8drhO-py3.10/bin/ask", line 6, in <module>
    sys.exit(run())
  File "/home/cesar/.cache/pypoetry/virtualenvs/autotx-qxA8drhO-py3.10/lib/python3.10/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
  File "/home/cesar/.cache/pypoetry/virtualenvs/autotx-qxA8drhO-py3.10/lib/python3.10/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
  File "/home/cesar/.cache/pypoetry/virtualenvs/autotx-qxA8drhO-py3.10/lib/python3.10/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/cesar/.cache/pypoetry/virtualenvs/autotx-qxA8drhO-py3.10/lib/python3.10/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
  File "/home/cesar/dev/polywrap/auto-tx/autotx/cli.py", line 121, in run
    result = autotx.run(cast(str, prompt), non_interactive)
  File "/home/cesar/dev/polywrap/auto-tx/autotx/AutoTx.py", line 87, in run
    result = self.try_run(prompt, non_interactive, summary_method)
  File "/home/cesar/dev/polywrap/auto-tx/autotx/AutoTx.py", line 217, in try_run
    float(chat.cost["usage_including_cached_inference"]["total_cost"]),
TypeError: tuple indices must be integers or slices, not str
```
seems that `chat.cost` is a tuple instead of a dictionary

Reply:
You need to run poetry install